cmd/greenhouse/main.go

```diff
@@ -127,20 +130,24 @@ func main() {
 	handleError(err, "unable to start manager")
 
 	// Register controllers.
-	for controllerName, hookFunc := range knownControllers {
-		if !isControllerEnabled(controllerName) {
-			setupLog.Info("skipping controller", "name", controllerName)
+	if !isWebhookMode(webhookOnlyModeEnv) {
```

Comment:
instead of an error which would fail to start, shall we have a fallback option? `regular` always? 

- In a production deployment this envs are not available at all, so the default will always be `regular`
- In local development it will mostly be `controllerOnly` as the default setup is run webhooks in cluster
- Only during local webhook development this poses an issue. We can always log the error and default it `controllerOnly`

What do you think @IvoGoman?

Reply:
for now I am erroring out as suggested

Reply:
I think it should be fine like this. If you do not configure anything then everything is run. In case both are set to be ignored, then it should error otherwise it might unintentionally start a second webhook server which goes unnoticed.